ext/nnstreamer/tensor_decoder/nnstreamer.fbs

```diff
@@ -0,0 +1,36 @@
+namespace NNStreamer;
+
+enum Tensor_type : int { 
+  NNS_INT32 = 0,
+  NNS_UINT32,
+  NNS_INT16,
+  NNS_UINT16,
+  NNS_INT8,
+  NNS_UINT8,
+  NNS_FLOAT64,
+  NNS_FLOAT32,
+  NNS_INT64,
+  NNS_UINT64 = 10,
```

Comment:
10? Is there any constraint which enforces that NNS_UINT64 should be 10? If not, I am not sure about this line.

Reply:
My bad.. I modified it so that it can be compatible with the existing enumeration of tensor_data.
Updated.